src/main/java/com/fasterxml/jackson/databind/ObjectReader.java

```diff
@@ -1202,6 +1218,25 @@ public void writeTree(JsonGenerator g, TreeNode tree) {
                 _considerFilter(_parserFactory.createParser(ctxt, f), false));
     }
 
+    /**
+     * Method that binds content read from given {@link Path}
+     * using configuration of this reader.
+     * Value return is either newly constructed, or root value that
+     * was specified with {@link #withValueToUpdate(Object)}.
+     *
+     * @param p Path that contains content to read
+     *
+     * @since 3.0
+     */
+    @SuppressWarnings("unchecked")
+    public <T> T readValue(Path p) throws JacksonException
```

Comment:
Will change `p` to `path` just because `p` traditionally refers to `JsonParser` (or whatever it'll get renamed as)